draft-ietf-tls-external-psk-importer.md

```diff
@@ -64,19 +64,19 @@ can be established via session tickets from prior connections or externally via
 mechanism. The protocol mandates that each PSK only be used with a single hash function.
 This was done to simplify protocol analysis. TLS 1.2 {{!RFC5246}}, in contrast,
 has no such requirement, as a PSK may be used with any hash algorithm and the
-TLS 1.2 PRF. While there is no known way in which the same external PSK might
-produce related output in TLS 1.3 and prior versions, only limited analysis has
-been done. Applications SHOULD provision separate PSKs for TLS 1.3 and prior
-versions when possible.
+TLS 1.2 Pseudorandom Function (PRF). While there is no known way in which the same
```

Comment:
Incredibly nitpicky nitpick: it seems RFC 5246 just writes "pseudorandom function" (all lowercase).

RFC 5869 likewise doesn't capitalize "key derivation function" when defining KDF. (RFC 8446 doesn't seem to ever expand it.)

Reply:
Thanks, stand by!